### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by IvanGoncharov**

### CODE REVIEW

**Review by n1ru4l — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -85,12 +85,15 @@
...
+    "ts-node": "10.9.1",
+    "typedoc": "0.23.10",
     "typescript": "4.7.4",
     "url-loader": "4.1.1"
   },
   "publishConfig": {
     "tag": "alpha"
+  },
+  "dependencies": {
+    "npm-check": "^6.0.1"
```
Comment (yaacovCR) [inline package.json:97]: Should this be a dev dependency or not included at all or is this intentional?

**Review by yaacovCR — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -85,12 +85,15 @@
...
   }
```
Comment (saihaj) [inline package.json:98]:
```suggestion
  }
```

**Review by saihaj — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (IvanGoncharov) [reply package.json:97]: @yaacovCR Good, catch.
I just forgot to add `--global` to a command.

File: package.json (hunk 2 above)
Comment (IvanGoncharov) [reply package.json]: @saihaj Thanks, good catch from you and @yaacovCR
Just forgot to add `--global`, fixed.

**Pull request merged**